spec/requests/stories/tagged_articles_spec.rb

```diff
@@ -85,6 +86,11 @@ def sets_nginx_headers
           expect { get "/t/#{tag.name}" }.to raise_error(ActiveRecord::RecordNotFound)
         end
 
+        it "renders not found if there are approved but scheduled posts" do
```

Comment:
Reading notes: this mimics the above `it "returns not found if no published posts and tag not supported"`